src/pages/vrf/v2/lootbox.mdx

```diff
@@ -0,0 +1,267 @@
+
+<Aside type="note">The deployer account must own all the tokens you want to distribute as rewards.</Aside>
+
+There's an example configuration file in [`scripts/data/tokens.json`](https://github.com/smartcontractkit/quickstarts-lootbox/blob/main/scripts/data/tokens.json) which includes a list of all supported token types:
+
+<Tabs client:visible>
+<Fragment slot="tab.1">ERC-20</Fragment>
+<Fragment slot="tab.2">ERC-721</Fragment>
+<Fragment slot="tab.3">ERC-1155</Fragment>
+<Fragment slot="panel.1">
+```json
+{
+  "tokenType": "ERC20",
+  "assetContract": "0x0000000000000000000000000000000000000001",
+  "totalAmount": "100000000000000000000",
+  "amountPerUnit": "10000000"
+}
+```
+</Fragment>
+<Fragment slot="panel.2">
+```json
+{
+  "tokenType": "ERC721",
+  "assetContract": "0x0000000000000000000000000000000000000002",
+  "tokenIds": ["1", "2"]
+}
+```
+</Fragment>
+<Fragment slot="panel.3">
+```json
+{
+  "tokenType": "ERC1155",
+  "assetContract": "0x0000000000000000000000000000000000000003",
+  "tokenId": "0",
+  "totalAmount": "100",
+  "amountPerUnit": "10"
+}
+```
+</Fragment>
+</Tabs>
+
+#### Whitelist
+
+The [Merkle tree](https://ethereum.org/en/developers/tutorials/merkle-proofs-for-offline-data-integrity/#introduction) for the private opening stage is generated from the address list in the [`scripts/data/whitelist.json`](https://github.com/smartcontractkit/quickstarts-lootbox/blob/main/scripts/data/whitelist.json) file. Edit the file and add all the addresses you want to list.
+
+If you don't want to do a private mint, leave the `whitelist.json` file empty, and the contract will be initialized in [public opening mode](#public-mode).
+
+## Run steps
+
+### Test
+
+- To run the unit tests, run the following command:
+
+  ```bash
+  npm run test
+  ```
+
+- If you want to see gas usage, run the following command:
+
+  ```bash
+  REPORT_GAS=true npm run test
+  ```
+
+- For coverage reports, run the following command:
+
+  ```bash
+  npm run coverage
+  ```
+
+### Deploy
+
+To run the deploy script, run the following command and replace `<network>` with the network you want to deploy to.
+
+```bash
+npx hardhat run scripts/deploy.js --network <network>
+```
+
+The network must be configured in [`hardhat.config.ts`](https://github.com/smartcontractkit/quickstarts-lootbox/blob/main/hardhat.config.ts).
+
+Besides deploying the contract, the [deploy script](https://github.com/smartcontractkit/quickstarts-lootbox/blob/main/scripts/deploy.ts) will also:
+
+1.  Approve each token configured in [`scripts/data/tokens.json`](https://github.com/smartcontractkit/quickstarts-lootbox/blob/main/scripts/data/tokens.json) for the deployed contract address.
+
+    This is a mandatory step because the contract must store the tokens in its own balance. So make sure the deployer account owns all the tokens you want to distribute as rewards.
```

Comment:
```suggestion
    This is a mandatory step because the contract must store the tokens in its own balance. It is important to ensure the deployer account owns all the tokens you want to distribute as rewards.
```